platform/core/Makefile

```diff
@@ -0,0 +1,53 @@
+export_dir_ios = export/ios
+ios_archs := aarch64-apple-ios aarch64-apple-ios-sim x86_64-apple-ios
+ios_targets := iPhone simulator
+build_type := $(if $(filter $(MODE), release),release,debug)
+
+build_ios_xcframework:
+# Ensure correct OS is used
+ifeq ($(OS),Windows_NT)
+	@echo "xcframework can be built only in macOS"
+	exit 1
+else
+ifneq ($(shell uname -s),Darwin)
+	@echo "xcframework can be built only in macOS"
+	exit 1
+endif
+endif
+
+	# Workspace setup
+	rm -rf $(export_dir_ios)
+	mkdir -p $(export_dir_ios)/iPhone
```

Comment:
Use a (mini) loop?

Reply:
Yeah, could be, based on `targets` lists.